kevm

```diff
@@ -108,17 +109,18 @@ run_web3_send() {
     }
     web3_params="$(join_args "$@")"
 
-    echo '{"jsonrpc": "2.0", "id": 1, "method": "'"$web3_method"'", "params": ['"$web3_params"']}' \
-        | netcat 127.0.0.1 "$web3_port" -q 0
+    curl -X POST --data '{"jsonrpc": "2.0", "id": 1, "method": "'"$web3_method"'", "params": ['"$web3_params"']}' 127.0.0.1:"$web3_port"
 }
 
 run_web3_ganache() {
     local web3_port
     web3_port="$1" ; shift
+    kevm_port=$(tests/web3/get_port.py)
+
 
     notif 'Launching web3 client'
-    $0 web3 "$web3_port" "$@" &
-    while ! netcat -z 127.0.0.1 "$web3_port"; do sleep 0.1; done
+    $0 web3 "$web3_port" "$kevm_port" "$@" &
+    while (! netcat -z 127.0.0.1 "$web3_port") || (! netcat -z 127.0.0.1 "$kevm_port"); do sleep 0.1; done
```

Comment:
`./kevm web3` doesn't take `kevm_port` as an argument anymore. So you'll have to only wait on `$web3_port` instead. Should be OK though I think.

Another option is to make `./kevm web3` take an _optional_ second argument, and if it's supplied then it's the `KPORT`, otherwise it generates one.